integration/Jenkinsfiles/utest_aurora.groovy

```diff
@@ -0,0 +1,88 @@
+/*
+ * (C) Copyright 2017 Nuxeo (http://nuxeo.com/) and others.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * Contributors:
+ *     atimic
+ */
+
+properties([
+        [$class: 'BuildDiscarderProperty', strategy: [$class: 'LogRotator', daysToKeepStr: '60', numToKeepStr: '60', artifactNumToKeepStr: '1']],
+        disableConcurrentBuilds(),
+        [$class: 'GithubProjectProperty', displayName: '', projectUrlStr: 'https://github.com/nuxeo/nuxeo/'],
+        [$class: 'RebuildSettings', autoRebuild: false, rebuildDisabled: false],
+        [$class: 'ParametersDefinitionProperty', parameterDefinitions: [
+                // branch hardcoded for now
+                [$class: 'StringParameterDefinition', name: 'BRANCH', description: '', defaultValue: 'feature-NXP-22077-add-aurora'],
+                // can't work in a pipeline multi-branch with build PR because BRANCH_NAME contains PR-XXXXX
+                //[$class: 'StringParameterDefinition', name: 'BRANCH', description: '', defaultValue: BRANCH_NAME],
```

Comment:
ask for help on this; give a look at https://github.com/nuxeo/nuxeo/pull/2110 which handle the two kinds of pipeline job

Reply:
In fact not really, #2110 handles pipeline multi-branch if the job is configured to build branch, but this is not the aim. We want to build only PRs.